### PR TITLE
fix makefile missing service env

### PR DESCRIPTION
**Pull request by filanov**
can you try to see if it works?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-SERVICE ?= quay.io/oamizur/introspector:latest
+SERVICE := $(or ${SERVICE},${SERVICE},quay.io/oamizur/introspector:latest)
```
Comment (ori-amizur) [inline Makefile]: Why twice ${SERVICE}

Comment (ori-amizur) [reply Makefile]: You need to remove one ${SERVICE}

Comment (filanov) [reply Makefile]: the first is a condition, the second is if's the condition is true and the third part is the else

Comment (filanov) [reply Makefile]: i think.. but it works

Comment (filanov) [reply Makefile]: you are right it works with two as well

**Pull request merged**